models/correlation_detector.cpp

```diff
@@ -346,3 +342,23 @@ nest::correlation_detector::handle( SpikeEvent& e )
     S_.incoming_[ sender ].insert( insert_pos, sp_i );
   } // device active
 }
+
+void
+nest::correlation_detector::calibrate_time( const TimeConverter& tc )
+{
+  if ( not P_.delta_tau_.is_step() )
+  {
+    std::string old = String::compose( "(was %1 ms)", P_.delta_tau_.get_ms() );
+    P_.delta_tau_ = 5 * Time::get_resolution();
+    std::string msg = String::compose( "Default value for delta_tau is now %1 ms %2", P_.delta_tau_.get_ms(), old );
```

Comment:
I find the message construction a bit difficult to follow. Why not store the old value in a double and then build the full string here?

Reply:
Agreed. Fixed in a62b0dabb.